docs/src/app_dev/daml_api/index.rst

```diff
@@ -41,6 +41,13 @@ Canton Network Token Standard APIs (CIP-0056)
 
 * See the reference docs for the :ref:`OpenAPI specifications of the Canton Network Token Standard <token_standard_openapi>`.
 
+See the following links for more information and code examples on how to use the Canton Network Token Standard:
```

Comment:
@OriolMunoz-da : please also add an entry to the release notes calling out the new docs. They are worth calling out :)